### PR TITLE
Use build script from ngeo

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by fredj** [comment]
@sbrunner can you please take a look ?

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -789,18 +789,15 @@ node_modules/ngeo/buildtools/mako_build.json: .build/node_modules.timestamp
...
-node_modules/.bin/closure-util$(NODE_BIN_POSTFIX): .build/node_modules.timestamp
-
 .PRECIOUS: .build/%.js
 .build/%.js: .build/%.json $(OL_JS_FILES) $(NGEO_JS_FILES) $(APP_JS_FILES) \
 		.build/gmftemplatecache.js \
 		.build/externs/angular-$(ANGULAR_VERSION).js \
 		.build/externs/angular-$(ANGULAR_VERSION)-q_templated.js \
 		.build/externs/angular-$(ANGULAR_VERSION)-http-promise_templated.js \
-		.build/externs/jquery-1.9.js \
-		node_modules/.bin/closure-util$(NODE_BIN_POSTFIX)
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Shouldn't we depends on  .build/node_modules.timestamp?

**Comment by sbrunner** [comment]
So small change :-), looks good, but just one comment, fix and merge :-)

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl (hunk 1 above)
Comment (fredj) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: added

**Comment by fredj** [comment]
Thanks for the review

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**